core/trino-main/src/main/java/io/trino/sql/planner/InputExtractor.java

```diff
@@ -83,6 +83,10 @@ public Set<Input> getInputs()
         @Override
         public Void visitTableScan(TableScanNode node, PlanFragmentId fragmentId)
```

Comment:
please squash the commits and follow the guideline https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Please help to review again, thanks